lib/cli/driver.js

```diff
@@ -262,9 +273,88 @@ class DriverCommand {
     return this.config.installedDrivers;
   }
 
-  update (/*{driver}*/) {
-    // TODO implementation
-    return {};
+  async update ({driver, force}) {
+    const all = driver === 'all';
```

Comment:
I'd rather move the magic word into constants

Reply:
also, it makes sense to call it `installed`

Reply:
ok makes sense